static/search.js

```diff
@@ -362,17 +366,17 @@ var Search = Control.extend({
 
 				} else {
 					// add “can-”, look for an exact match in the title field, and apply a positive boost
-					q.term('can-' + searchTerm, { usePipeline: false, fields: ['title'], boost: 12 });
+					q.term('can-' + searchTerm, { usePipeline: false, boost: 12 });
```

Comment:
I removed the `fields` property when we were testing things, but didn’t mean for this change to be committed. If it’s working then ok, but it might be useful to add back.